src/heads-up/views/slack.vue

```diff
@@ -25,13 +26,62 @@
 
     <panel-section title="Properties">
       <properties-list>
-        <template v-if="og.title">
-          <dt>og:title</dt>
+        <template>
+          <dt>
+            <p v-if="!og.title">
+              og:title
+            </p>
+            <app-tooltip
+              class="properties-item__tooltip"
+              placement="bottom-start"
+            >
+              <div v-if="og.title">
+                og:title
+              </div>
+              <div v-else>
+                <InfoIcon class="properties-item__icon" />
+              </div>
+              <template v-slot:info>
+                <property-data
+                  type="og:title"
+                  :exist="tooltip.title.exist"
+                  :tag="tooltip.title.tag"
+                  :value="tooltip.title.content"
+                />
+              </template>
+            </app-tooltip>
+          </dt>
           <dd>{{ og.title }}</dd>
         </template>
         <template v-if="og.image">
-          <dt>og:image</dt>
-          <dd>
+          <dt>
+            <p v-if="og.image === null">
+              og:image
+            </p>
+            <app-tooltip
+              v-if="showTooltip"
```

Comment:
This may need to be more specific, since there are three tooltips in this file. Maybe call it `showImageTooltip`?